library/core/src/mem/transmutability.rs

```diff
@@ -0,0 +1,39 @@
+/// Are values of a type transmutable into values of another type?
+///
+/// This trait is implemented on-the-fly by the compiler for types `Src` and `Self` when the bits of
+/// any value of type `Self` are safely transmutable into a value of type `Dst`, in a given `Context`,
+/// notwithstanding whatever safety checks you have asked the compiler to [`Assume`] are satisfied.
+#[unstable(feature = "transmutability", issue = "none")]
```

Comment:
Create a tracking issue and reference it everywhere

Reply:
Resolved by https://github.com/rust-lang/rust/pull/92268/commits/fe8f757e12f11943baccfd59b43a5166a9fe87ab